src/Schema/AbstractIndex.php

```diff
@@ -123,7 +123,13 @@ public function sqlStatement(DriverInterface $driver, bool $includeTable = true)
         }
 
         //Wrapping column names
-        $columns = implode(', ', array_map([$driver, 'identifier'], $this->columns));
+        $columns = [];
+        foreach ($this->columns as $column => $expression) {
+            $columns[] = is_int($column)
```

Comment:
see, now we have to detect this column storage format used, this makes this code harder to understand